queries/hyprlang/highlights.scm

```diff
@@ -0,0 +1,65 @@
+(comment) @comment
+
+(source "source" @keyword)
+
+(exec "exec" @keyword)
+(exec "exec-once" @keyword)
+
+(keyword
+  (name) @keyword
+)
+
+(assignment
+  (name) @property
+)
+
+(section
+  (name) @namespace
+)
+
+(section
+  device: (name) @type
+)
+
+(variable) @variable
+(variable "$" @punctuation.special)
+
+(number) @number
+
+(boolean) @boolean
+
+(mod) @constant
+
+(vec2
+  (number) @number
+  (number) @number
```

Comment:
One should be enough. This creates n^2 captures for n number nodes

Reply:
In fact, this pattern is not needed at all, since the capture in line 27 already covers this.

Reply:
You're right, I fixed it